docs/tutorials/01_basics/05_control_flow.md

```diff
@@ -0,0 +1,202 @@
+# Control Flow
+
+## Primitive Types
+
+### Nil
```

Comment:
A note regarding nil not been a value of String and other reference types will make sense here.

As a motivational example we could use finding the occurrence of a string inside another, which could return nil when not present. This could replace the car analogy.